app/views/posts/index.js.erb

```diff
@@ -1 +1,6 @@
-<%= infinite_append ".list-post ul", @posts %>
+$('#posts').append('<%= j render(@posts) %>');
```

Comment:
can we try and not add ruby code in the js file, and implement this without the ruby var being used in js.